### PR TITLE
sci-electronics/kicad: Bump to stable 4.0.0 release

### PR DESCRIPTION
**Pull request by zpuskas**
@idella 
@jlec 
@gentoo/proxy-maint

### CODE REVIEW

File: sci-electronics/kicad/kicad-4.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+RDEPEND="${CDEPEND}
+	sci-electronics/electronics-menu"
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# remove all the non unix file endings
+	find "${S}" -type f -name "*.desktop" | xargs -n1 dos2unix || die "dos2unix failed"
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.0.ebuild]: You need to use ```assert``` here and not ```die```

File: sci-electronics/kicad/kicad-4.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+		# remove additional demos/examples as its not strictly required to run the binaries 
+		sed -e '/add_subdirectory( demos )/d' -i CMakeLists.txt || die "sed failed"
+	fi
+
+	# Add important missing doc files
+	sed -e 's/INSTALL.txt/AUTHORS.txt CHANGELOG.txt README.txt TODO.txt/' -i CMakeLists.txt || die "sed failed"
+}
+
+src_configure() {
+	need-wxwidgets unicode
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.0.ebuild]: Is this still needed? I vaguely remember that there is a QA warning about deprecation. But I might be wrong.

Comment (zpuskas) [inline sci-electronics/kicad/kicad-4.0.0.ebuild]: I did not see any warnings for this. Also packages like dev-python/wxpython/wxpython have it so I guess it's well in use.

**Comment by idella** [comment]
merged

**Pull request merged**